templates/taxi/car_form.html

```diff
@@ -0,0 +1,11 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+    <h1>{{ object|yesno:"Update,Create" }} Car</h1>
```

Comment:
The use of the `yesno` filter with `object|yesno:"Update,Create"` may not work as intended because `object` is not a boolean value. Consider using a condition to check if the object exists and then display 'Update Car' or 'Create Car' accordingly.